models/sources/src_history_ledgers.yml

```diff
@@ -172,3 +172,10 @@ sources:
               - not_null:
                   config:
                     where: closed_at > current_timestamp - interval 2 day
+
+          - name: soroban_fee_write_1kb
+            description: '{{ doc("soroban_fee_write_1kb") }}'
+            tests:
+              - not_null:
```

Comment:
Double checking; can this be null? Idonno if it gets set for non soroban

Reply:
Ran a Null Check across the full dataset on raw. There are no Null records

select
Null_Check,
count(*) as rec_count
from
(
SELECT 
case when soroban_fee_write_1kb is null then "Null"
else "Not Null"
end as Null_Check
FROM `crypto-stellar.crypto_stellar.history_ledgers` 
--WHERE TIMESTAMP_TRUNC(closed_at, MONTH) = TIMESTAMP("2024-01-01")
)
group by
Null_Check
;